src/domain/child/queries/ChildQueries.ts

```diff
@@ -40,6 +40,63 @@ export const childByIdQuery = gql`
       lastName
       birthdate
       postalCode
+      occurrences {
+        edges {
+          node {
+            event {
+              id
+              translations {
+                name
+                shortDescription
+              }
+            }
+          }
+        }
+      }
+      availableEvents {
+        edges {
+          node {
+            id
+            translations {
```

Comment:
you can get the name directly here if you have an up to date backend
No need to fiddle with translations!